class AuthException(Exception):
    pass


class UserInitializationFailure(AuthException):
    pass
